Another mock change

MatthewJWalls opened the pull request:
## What does this change?

## What is the value of this and can you measure success?

## Does this affect other platforms - Amp, Apps, etc?

## Screenshots

<!-- AB test? https://git.io/v1V0x -->
<!-- AMP question? https://git.io/v1V0p -->
<!-- Does this PR meet the contributing guidelines? https://git.io/v1VEJ -->

MatthewJWalls commented:
PR build results: 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/mobile.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/wide.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/tablet.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/desktop.png

PRBuilds commented:
PR build results: 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-1/screenshots/mobile.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-1/screenshots/wide.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-1/screenshots/tablet.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-1/screenshots/desktop.png 

 -automated message

PRBuilds commented:
PR build results:
> [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
)•[wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
)•[tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
)•[desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
 -automated message

PRBuilds commented:
PR build results:
> [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%2F2014%2Fmay%2F21%2Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)

 -automated message

PRBuilds commented:
PR build results:
> [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)

 -automated message

PRBuilds commented:
PR build results:
> [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)

 -automated message

PRBuilds commented:
PR build results:
> [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [thrown-exceptions.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.txt 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [thrown-exceptions.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.txt 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
) • [performanceTestResults.csv](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceTestResults.csv 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [thrown-exceptions.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.txt 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > 

 -automated message

PRBuilds commented:
PR build results:
screenshots > 
other > [thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
)

 -automated message

MatthewJWalls commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
)

 -automated message

PRBuilds commented:
PR build results:
screenshots > [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png 
) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png 
) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png 
) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png 
)
other > [thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js 
) • [performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt 
)

 -automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)
    
-automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)
    
-automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)
    
-automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)
    
-automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)
    
-automated message

MatthewJWalls commented:
PR build results:

screenshots
[wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results

    
-automated message

MatthewJWalls commented:
PR build results:

screenshots
[wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest results

    
-automated message

PRBuilds commented:
PR build results:

screenshots
[mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest (1)
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)

-automated message

PRBuilds commented:
PR build results:

screenshots
[desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png)
    
exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    

webpagetest (0)
[performanceComparisonSummary.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/performanceComparisonSummary.txt)

-automated message

MatthewJWalls commented:
PR build results:


screenshots
[wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)



exceptions (2)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)    




-automated message

MatthewJWalls commented:
PRBuilds failed when building this PR: global name 'reporter' is not defined

PRBuilds commented:
PR build results:


screenshots
[wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png)



exceptions (0)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/thrown-exceptions.js)




-automated message

GHaberis committed:
remove chain from anagram-helper

GHaberis committed:
fix unit test

GHaberis committed:
remove chain from clues

GHaberis committed:
remove chain from crossword helpers

GHaberis committed:
remove chain from comment-count

GHaberis committed:
remove chain from lightbox

SiAdcock committed:
move projects/common/modules/experiments/tests/tailor-survey.js from legacy to standard JS

SiAdcock committed:
transform projects/common/modules/experiments/tests/tailor-survey.js to ES6 module

jranks123 committed:
Change ask on engagement banner to weekly rather than monthly price

janua committed:
Fix last parameter in es5to6

janua committed:
transform bootstraps/commercial.js to ES6 module

Commit:
Merge pull request #16564 from guardian/ad-call-targeting-variables

Ad call targeting variables for Switch

Commit:
Merge pull request #16563 from guardian/the-snap-email

SNAP

Commit:
add branding to paid-for content in rich links

davidfurey committed:
Update exampleUrl for flyer email

lps88 committed:
Merge pull request #16560 from guardian/paid-rich-links

Paid content in rich links

janua committed:
Fix lint before retry of lebab

jranks123 committed:
initial commit

kelvin-chappell committed:
Add hardcoded logo to football feed pages

janua committed:
Fix commercial bootstrap to use arrow function as default export

GHaberis committed:
remove chain from onward/history

GHaberis committed:
remove chain from onwards/search-tool

davidfurey committed:
Merge pull request #16562 from guardian/disable-flyer-test

Remove flyer email a/b test

janua committed:
Add in some types to the commercial bootstrap

GHaberis committed:
remove chain from live-blog-updates.js

sndrs committed:
cahnge node type

sndrs committed:
existence of node is implied by typing

GHaberis committed:
remove chain

sndrs committed:
defaults for options

sndrs committed:
genericise add-event-listener module

SiAdcock committed:
linting

sndrs committed:
better import name

SiAdcock committed:
export tailor survey constructor function

JustinPinner committed:
Enable test from desktop BP. Next is to drop to tablet.

SiAdcock committed:
tailor is a class

JustinPinner committed:
Merge branch 'master' into jp-remove-ab-epic-to-support-landing-page-switch

# Conflicts:
#	common/app/conf/switches/ABTestSwitches.scala

JustinPinner committed:
Lydia's switch got removed by IntelliJ's resolve conflicts. Put it back.

janua committed:
Merge remote-tracking branch 'origin/master'

GHaberis committed:
formatting

janua committed:
Merge branch 'master' into es6-bootstraps_commercial-0591

janua committed:
Remove comment

joelochlann committed:
Add campaign code suffix to ophan events as well (#16570)

joelochlann committed:
Revert change which seems to have broken master (#16572)

SiAdcock committed:
move helper functions outside class

GHaberis committed:
remove chain from es5to6

JustinPinner committed:
Merge pull request #16559 from guardian/jp-remove-ab-epic-to-support-landing-page-switch

Remove epic to support landing page AB test and switch

Commit:
Extend preflight ad call switch

Commit:
Merge pull request #16574 from guardian/extend-preflight-switch

Extend preflight ad call switch

Commit:
extend the paidcontent v outbrain test

Commit:
remove paid card logo test

SiAdcock committed:
refactor imports

NataliaLKB committed:
Adding a placeholder element for AMP

sndrs committed:
Merge branch 'master' of github.com:guardian/frontend into es6-lib_add-event-listener

SiAdcock committed:
type safety

sndrs committed:
Merge pull request #16532 from guardian/es6-lib_add-event-listener

ES6 lib/add-event-listener

JustinPinner committed:
Enable from tablet breakpoint.

sndrs committed:
move bootstraps/admin.js from legacy to standard JS

sndrs committed:
transform bootstraps/admin.js to ES6 module

sndrs committed:
transform bootstraps/admin.js content to ES6

SiAdcock committed:
more safety

sndrs committed:
fix lint errors with bootstraps/admin.js after transform to ES6

Commit:
remove paid card logo test variant

Commit:
fix switch name to match test id!

SiAdcock committed:
reinstate bean

Commit:
typo fix

joelochlann committed:
Give myself more relevant es6 conversions (#16571)

* Give myself more relevant es6 conversions

* Remove es5to6-files-per-user.js

joelochlann committed:
Fix epic in liveblog campaign code (#16576)

* Add campaign code suffix to ophan for impressions tracking as well as to the links for conversion tracking
* When converting page id to a campaign code, turn / into __ and - into _, rather than both into _
* Fix test which fails because pageId isn't mocked in the test framework

lps88 committed:
Merge pull request #16577 from guardian/switches

Switches etc

Commit:
extends acquisition tests whose expiry dates fall on bank holiday Monday

Commit:
initial commit

janua committed:
Merge branch 'es6-bootstraps_commercial-0591'

NataliaLKB committed:
Merge pull request #16580 from guardian/nb-placeholder-amp

Adding a placeholder element for AMP

GHaberis committed:
reduce margin-top on youtube atom on mobile

TBonnin committed:
Update facia architecture documentation

sndrs committed:
oopsie

SiAdcock committed:
Merge pull request #16573 from guardian/es6-projects_common_modules_experiments_tests_tailor-survey

ES6ify tailor-survey

jfsoul committed:
Grammar corrections

jfsoul committed:
More grammar

lps88 committed:
Disable branding for paid cards (#16581)

* stop branding appearing on all paid cards

* disable paid cards branding

* disable paid cards branding and remove code stuffs

Commit:
move projects/common/modules/video/video-container.js from legacy to standard JS

Commit:
transform projects/common/modules/video/video-container.js to ES6 module

Commit:
transform projects/common/modules/video/video-container.js content to ES6

Commit:
Mechanical fix of eslint errors

Mullefa committed:
Merge pull request #16578 from guardian/acq-typo-fix

acquisition test selector typo fix

oilnam committed:
Extend Tailor AB test

janua committed:
move projects/admin/bootstraps/commercial-browser-performance.js from legacy to standard JS

janua committed:
transform projects/admin/bootstraps/commercial-browser-performance.js to ES6 module

janua committed:
transform projects/admin/bootstraps/commercial-browser-performance.js content to ES6

kelvin-chappell committed:
Use logo with smaller file size

kelvin-chappell committed:
Refactor

kelvin-chappell committed:
Replace def with val

kelvin-chappell committed:
Replace val with def

kelvin-chappell committed:
Merge branch 'master' into kc-football-logo

Commit:
Add `validate-a11y` task

Commit:
Add pa11y dependency

Mullefa committed:
Merge pull request #16579 from guardian/acq-test-extensions

extends acquisition tests whose expiry dates fall on bank holiday Monday

Commit:
Add task to makefile

sndrs committed:
Merge pull request #16575 from guardian/es6-bootstraps_admin

ES6 bootstraps/admin

kelvin-chappell committed:
Merge pull request #16582 from guardian/kc-football-logo

Add hardcoded logo to football feed pages

JustinPinner committed:
Set expected test audience and check for cookie before running.

oilnam committed:
Merge pull request #16586 from guardian/oilnam-extend-tailor-1

Extend Tailor AB test

JustinPinner committed:
Merge branch 'master' into jp-ab-bundle-price-test-1

# Conflicts:
#	static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

oilnam committed:
Extend Tailor test

Commit:
all copy added

oilnam committed:
Merge pull request #16589 from guardian/oilnam-tailor-ext2

Extend Tailor test

Commit:
Add types and null checks

Commit:
Add types for all top-level functions

GHaberis committed:
fix bug in onwards/search-tool

Commit:
Obey linting

Commit:
Removes Skiing from UK, US, AU, INT editions navs (#16590)

* Removes Skiing from UK, US, AU, INT editions navs

* Also remove it from the newNav

Commit:
Use correct Flow function type syntax

kelvin-chappell committed:
Fix logo not appearing on /fixtures page

TBonnin committed:
Fix typos and grammar

kelvin-chappell committed:
Merge pull request #16592 from guardian/kc-fix-fixtures

Fix logo not appearing on /fixtures page

TBonnin committed:
Add info about Front cron update freqency

Commit:
most of the work done apart from quotation mark

janua committed:
fix lint errors with projects/admin/bootstraps/commercial-browser-performance.js after transform to ES6

TBonnin committed:
Add info about draft/live fronts

GHaberis committed:
Merge pull request #16569 from guardian/gh-remove-chain

remove lib/chain

NataliaLKB committed:
content meta explore template rewrite

NataliaLKB committed:
removing courses, which we don't market anymore

gidsg committed:
add class to main media atom

janua committed:
Fix call to toFixed; needs to be a number not an Array<number>

gidsg committed:
move code up

janua committed:
Declare globals for flowtype

NataliaLKB committed:
Merge pull request #16594 from guardian/nb-remove-courses

Removing courses from the mobile nav, which we don't market anymore

NataliaLKB committed:
adding temp hack for interactives and crosswords to keep the padding

gidsg committed:
add class specific to media atom

JustinPinner committed:
Merge pull request #16588 from guardian/jp-ab-bundle-price-test-1

Bundle Subscription Price Test 1 (AB)

NataliaLKB committed:
Merge pull request #16596 from guardian/nb-margin-hack-interactives

Adding temp hack for interactives and crosswords to keep the padding

Commit:
background image used for quotation mark following the technique here #16132

NataliaLKB committed:
Merge pull request #16593 from guardian/nb-content-meta-explore

Content meta explore template rewrite

jranks123 committed:
set up engagement banner tests

Commit:
changes based on Iona's comments

sndrs committed:
blh

TBonnin committed:
Merge pull request #16585 from guardian/tb-facia-doc

Update facia architecture documentation

sndrs committed:
blh

sndrs committed:
silly

sndrs committed:
update precommit hook to js, and lint on commit

sndrs committed:
this should get through with proper linting

sndrs committed:
break commit checks into sections

sndrs committed:
didn’t mean to commit thiese

sndrs committed:
nor this

katebee committed:
Add Morning Briefing to the newsletters list

katebee committed:
Make the Long Read teaser... less long, and stop layout issue

sndrs committed:
Merge branch 'master' of github.com:guardian/frontend into lint-staged

sndrs committed:
update js recomendations

sndrs committed:
Merge pull request #16599 from guardian/lint-staged

Lint (and fix) JS when committing

sndrs committed:
Merge branch 'master' of github.com:guardian/frontend into no-old-js

# Conflicts:
#	static/src/javascripts/DO_NOT_ADD_NEW_JS_HERE.md

Commit:
Add email preferences link to privacy tab

sndrs committed:
Merge pull request #16602 from guardian/no-old-js

Update js recomendations

janua committed:
Add types to commercial-browser-performance

janua committed:
Merge branch 'es6-projects_admin_bootstraps_commercial-browser-performance'

janua committed:
Merge branch 'master' into commercial-browser-performance-types

Commit:
Rewording

Commit:
Merge pull request #16603 from guardian/add-email-prefs-link-to-privacy-tab

Add email preferences link to privacy tab

Commit:
most pull request comments addressed

GHaberis committed:
refactor css for youtube atoms

GHaberis committed:
css validation fix

Commit:
extend switch

katebee committed:
Merge branch 'master' into kw-020517-morningmail

GHaberis committed:
remove comment

GHaberis committed:
refactor

lps88 committed:
Merge pull request #16607 from guardian/extend-a-switch

Extend switch

jranks123 committed:
clean up

Commit:
Merge pull request #16587 from guardian/gp-pa11y-integration

Add automated accessibility checking

Commit:
Bootstrap StoryQuestions embed

Commit:
Add routes

Commit:
Model of a story question page

jranks123 committed:
Merge pull request #16598 from guardian/paywallAndPaypalEBTest

Engagement Banner Paywall and Paypal Test

GHaberis committed:
nest pseudo selectors

SiAdcock committed:
convert tailor

GHaberis committed:
Merge pull request #16584 from guardian/gh-main-media-youtube-atom-on-mobile

reduce margin-top on youtube atom on mobile

SiAdcock committed:
no lint deleted files on commit

katebee committed:
Updates to the email listings and example URLs

Commit:
more work after help from Gustav and Simon

katebee committed:
Merge pull request #16604 from guardian/kw-020517-morningmail

Add Morning Briefing newsletter, and fix copy length

Commit:
unused svg input removed

SiAdcock committed:
Merge pull request #16608 from guardian/sa-no-lint-deleted-files

Don't lint deleted files on commit

Commit:
Remove unused use case

Commit:
Fix paths

Commit:
Style

Commit:
Fill embed page

Commit:
force inline fluid ads to have the correct margins

SiAdcock committed:
import named export

Commit:
undo unintended change!

SiAdcock committed:
tailor methods return promises

sndrs committed:
--no-verify first step of eslint conversion

sndrs committed:
change slack room

sndrs committed:
add wiki link

Commit:
Add a comment expressing scepticsm

Commit:
Merge pull request #16591 from guardian/es6-projects_common_modules_video_video-container

Es6 projects common modules video video container

SiAdcock committed:
update comment to reflect true return type

sndrs committed:
Merge pull request #16614 from guardian/es-conversion-lint-optout

 --no-verify first step of eslint conversion

sndrs committed:
update prettier

SiAdcock committed:
remove superfluous jsdoc style comments

Commit:
move projects/common/modules/video/videojs-options.js from legacy to standard JS

sndrs committed:
Merge pull request #16615 from guardian/prettier-1.3.0

update prettier to 1.3.0

janua committed:
Merge branch 'commercial-browser-performance-types'

SiAdcock committed:
Extend Explore test

janua committed:
move projects/commercial/modules/ad-sizes.js from legacy to standard JS

janua committed:
transform projects/commercial/modules/ad-sizes.js to ES6 module

katebee committed:
Merge pull request #16609 from guardian/kw-170502-emails

Updates to the email listings and example URLs

Commit:
Use a default argument and an explicit return type

janua committed:
Add types to ad-sizes

Commit:
changes based off @ionamckendrick's feedback

lps88 committed:
Merge pull request #16611 from guardian/inline-fluid

Fix margins of inline fluid ads

Commit:
Add Atom embedded stylesheet

Commit:
Merge pull request #16617 from guardian/es6-projects_common_modules_video_videojs-options

ES6ify videojs-options

SiAdcock committed:
Merge pull request #16616 from guardian/SiAdcock-patch-1

Extend Explore test

Commit:
move projects/common/views/svg.js from legacy to standard JS

Commit:
transform projects/common/views/svg.js to ES6 module

Commit:
Merge pull request #16610 from guardian/rk-story-question-atoms

Story questions embed

Commit:
Add types and use string interpolation

Commit:
changes based on @gustavpursche's feedback (apart from BEM naming convention)

Commit:
Revert #16526 as it broke username changes

Commit:
Merge pull request #16619 from guardian/revert-16526

Revert #16526 as it broke username changes

janua committed:
Change to Object type as this syntax isn't yet supported by babel plugin

jfsoul committed:
Strip invalid XML characters from image credits

Commit:
BEM naming convention used

sndrs committed:
add ‘get’ method ot config object

sndrs committed:
comments

sndrs committed:
Merge pull request #16621 from guardian/config-get-method

add ‘get’ method to config object

Mullefa committed:
Merge pull request #16597 from guardian/acq-epic-testimonials

Epic testimonials

jfsoul committed:
Merge pull request #16620 from guardian/jfs-strip-credits

Strip invalid XML characters from image credits

Commit:
Extend navigation more toggle and fix tracking

Commit:
Go with the flow

Commit:
Use a non-default export and add a spec

Commit:
Merge pull request #16624 from guardian/gp-navigation-more-toggle-take1000

Navigation more toggle - fix tracking

SiAdcock committed:
Merge pull request #16612 from guardian/es6-projects_common_modules_tailor_tailor

ES6 tailor

JustinPinner committed:
Change thrasher copy to make more relevant to test offering.

Commit:
Suggest the liberating power of constraints

kant committed:
Minor fix

Commit:
Doc: Fix minor misspelling

Minor fix

janua committed:
Merge branch 'es6-projects_commercial_modules_ad-sizes-2143'

JustinPinner committed:
Capture a new audience segment for this thrasher test iteration

JustinPinner committed:
Merge pull request #16625 from guardian/jp-ab-bundle-price-test-1a

Modify AB test to change thrasher copy

Commit:
Merge pull request #16623 from guardian/es6-projects_common_views_svg

Es6 projects common views svg

joelochlann committed:
Fix AMP docs link in PR template (#16628)

Commit:
Add username instructions to edit profile

Commit:
Merge pull request #16630 from guardian/add-username-instructions-to-edit-profile

Add username instructions to edit profile

jranks123 committed:
engagement banner paypal and paywall test round 2

joelochlann committed:
Make epic in liveblog pass AMP validation (#16629)

Fixes an AMP validation issue in #16519.

Items in an amp-live-list are required to have an id and data-sort-time.
https://www.ampproject.org/docs/reference/components/amp-live-list#items

I was inserting a placeholder element for the epic which didn't have these. So I've opted to ditch the placeholder element and instead add a class to the block itself, then insert the epic after that selector.

sndrs committed:
remove gareth from es6 file list

jranks123 committed:
make joe's reccomended css changes

jranks123 committed:
Merge pull request #16631 from guardian/engagementRound2paypal

Engagement banner paypal and paywall test round 2

jranks123 committed:
change campaign code for round two of eb test

jranks123 committed:
Merge pull request #16635 from guardian/engagementRound2paypal

Change campaign code prefix for round two of engagement banner test

Commit:
Merge branch 'master' of github.com:guardian/frontend

Commit:
Include new test

PRBuilds commented:
PRbuilds results:









--automated message

PRBuilds commented:
PRbuilds results:










**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/loadtesting.txt)


    
--automated message

PRBuilds commented:
PRbuilds results:










**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/loadtesting.txt)


    
--automated message

PRBuilds commented:
PRbuilds results:










**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/loadtesting.txt)


    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (0)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)



**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/loadtesting.txt)


    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (8)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)



**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/loadtesting.txt)


    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (0)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)




    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (7)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)





**Microdata Validation**
failed: 0 failures

    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (8)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)





**Microdata Validation**
failed: 0 failures

    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (7)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)





**Microdata Validation** (0)
[microdata.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/microdata.txt)

    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (8)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)





**Microdata Validation** (0)
[microdata.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/microdata.txt)

    
--automated message

PRBuilds commented:
PRbuilds results:








**A11y validation** (8)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/a11y-report.txt)





**Microdata Validation** (1)
[microdata.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-2/microdata.txt)

    
--automated message
